node_task3/userRouter/getUserRouter.js

```diff
@@ -0,0 +1,18 @@
+const { Router } = require("express");
+const Service = require("../userService/services");
+const service = new Service();
+const getRouter = Router();
+getRouter.get("/:id", async (req, res) => {
+  const userQueryData = await service.get(req.params);
+  const msg = "user data not found";
```

Comment:
Please move the string literals to the constants.js file, export and use them as needed.

const USER_NOT_FOUND = "User data not found";

module.exports = {USER_NOT_FOUND}

Reply:
implemented